src/main/resources/dbreplica/migration/V1.0.27__reportable_task_plsql.sql

```diff
@@ -0,0 +1,323 @@
+          RAISE INFO 'Insert reportable task record for : %', l_task_id;
+        else
+          RAISE WARNING '% : Task with an incomplete history and will therefore not be reported on.', l_task_id;
+          EXIT;
+        end if;
+
+       if (l_update_action='AutoAssign') and (l_state='ASSIGNED') then
+            l_first_assigned_date = l_updated::date;
+            l_first_assigned_date_time = l_updated;
+            l_wait_time_days = 0;
+            l_wait_time = 0;
+            l_number_of_reassignments = 0;
+       end if;
+
+        insert into cft_task_db.reportable_task
+            (task_id, task_name, task_type, due_date_time,
+             state, task_system, security_classification,
+             title, description, notes, major_priority,
+             minor_priority, assignee, auto_assigned,
+             execution_type_code, work_type, role_category,
+             has_warnings, assignment_expiry, case_id,
+             case_type_id, case_category, case_name,
+             jurisdiction, region, region_name,
+             location, location_name, business_context,
+             termination_reason, created, updated_by, updated,
+             update_action, created_date, due_date, last_updated_date,
+             first_assigned_date, first_assigned_date_time, wait_time_days,
+             wait_time, number_of_reassignments, state_label,
+             role_category_label, jurisdiction_label, case_type_label,
+             additional_properties, reconfigure_request_time, next_hearing_id,
+             next_hearing_date, priority_date, last_reconfiguration_time,
+             report_refresh_time)
+            values
+              (l_task_id, l_task_name, l_task_type, l_due_date_time,
+              l_state, l_task_system, l_security_classification,
+              l_title, l_description, l_notes, l_major_priority,
+              l_minor_priority, l_assignee, l_auto_assigned,
+              l_execution_type_code, l_work_type, l_role_category,
+              l_has_warnings, l_assignment_expiry, l_case_id,
+              l_case_type_id, l_case_category, l_case_name,
+              l_jurisdiction, l_region, l_region_name,
+              l_location, l_location_name, l_business_context,
+              l_termination_reason, l_created, l_updated_by, l_updated,
+              l_update_action, l_created::DATE, l_due_date_time::DATE,
+              l_updated::DATE, l_first_assigned_date, l_first_assigned_date_time,
+              l_wait_time_days, l_wait_time, l_number_of_reassignments,
+              l_state_label, l_role_category_label, l_jurisdiction_label,
+              l_case_type_label, l_additional_properties, l_reconfigure_request_time,
+              l_next_hearing_id, l_next_hearing_date, l_priority_date,
+              l_last_reconfiguration_time, current_timestamp)
+        returning update_id into l_update_id;
+        l_new_task = false;
+    else
+        RAISE INFO 'Attempting to update task_id : %', l_task_id;
```

Comment:
To keep it in line with the other log messages:

```suggestion
        RAISE INFO 'Updating reportable task record for task_id : %', l_task_id;
```